lib/src/browser/disposable_browser.dart

```diff
@@ -158,6 +161,11 @@ class Disposable implements disposable_common.Disposable {
   }
 
   @override
+  void flagLeak([String description]) {
```

Comment:
since we proxy to an internal disposable object, this method will never be called by `_disposable.dispose()`. Does it make sense to provide some sort of hook in the `_InnerDisposable` class like we did for the `onDispose` method? If we don't do something the parent class of the `LeakFlag` in the heap trace will always be `_InnerDisposable`.